.coderabbit.yml

```
@@ -0,0 +1,54 @@
+
```

YAMLlint (1.37.1)

[error] too many blank lines

(1 > 0) (empty-lines)

---

[error] wrong new line character: expected \n

(new-lines)